src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -450,11 +450,7 @@ function setUsage(
         } else if (usages.roundTrips.includes(name)) {
             m.Usage = Usage.RoundTrip;
         } else {
-            if ((m as InputEnumType).IsExtensible) {
-                m.Usage = Usage.RoundTrip;
-            } else {
-                m.Usage = Usage.None;
-            }
+            m.Usage = Usage.None;
```

Comment:
I think we need to report to CADL team to have a usage calculation here.

Reply:
No. cadl compiler correctly resolve the usage for enum. We used to hard-code extensible enum to RoundTrip as workround. Now we will resolve the enum issue, and this workround need to be removed.